File: papi/lista1/exercicio3.py
a = 0
b = 0
c = 0

while True:
    try:
        a = int(input("Digite o valor 'a': "))
        break
    except ValueError:
        print("Por favor digite um número")

while True:
    try:
        b = int(input("Digite o valor 'b': "))
        break
    except ValueError:
        print("Por favor digite um número")

if a == b:
    print("'a' é igual a 'b'")
    print("a soma dos valores é:", a + b)
else:
    print("'a' é diferente de 'b'")
    print("o resultado da multiplicação dos valores 'a' e 'b' é:", a * b)

File: papi/lista1/exercicio3.kt
package lista1

fun main() {
    var a: Double
    var b: Double
    var c: Double

    while (true) {
        try {
            print("Por favor digite o primeiro número: ")
            val input = readLine()
            a = input!!.toDouble()
            break
        } catch (e: NumberFormatException) {
            println("Por favor digite um número vaĺido, com o ponto como separador decimel")
        }
    }

    while (true) {
        try {
            print("Por favor digite o segundo número: ")
            val input = readLine()
            b = input!!.toDouble()
            break
        } catch (e: NumberFormatException) {
            println("Por favor digite um número vaĺido, com o ponto como separador decimel")
        }
    }

    if (a == b) {
        println("Os números são iguais")
        print("A soma deles é: ")
        c = a + b
    } else {
        println("Os números são diferentes")
        print("A multiplicação entre eles é: ")
        c = a * b
    }
    println(c)

}